docs/proposals/organizations.md

```diff
@@ -0,0 +1,124 @@
+# Organizations
+
+## Problems
+ 1.  Cluster administrators want to allocate a pool of resources to a subset of their company, and delegate the authority to manage how that pool is assigned to an organization administrator.
+ 2.  An organization administrator needs to be able to limit a particular project's usage of shared quota (think htb rate/ceil for tc qdisc classes).
+ 3.  Cluster administrators want to delegate control over groups of users to an organization administrator - that organization administrator should be able to manage the access of a set of users to individual projects under that organization umbrella.
+ 4.  Cluster administrators want to allow self-service of users on the cluster, but the total allocated resources those self service users can get access to is limited (to prevent abuse / unfair use of resources).
+
+We can do this with an Organization entity that can manage multiple projects.  An Organization can have multiple org-owners who are allowed to manage quota allocation to owned projects and OrgGroups (groups that scoped to projects owned by an Organization).  A project is owned by at most one Organization, an Organization can own muliple projects.
+
+### Open Questions
+ 1.  Should we have a two layer scoping `/api/v1/organization/<org name>/namespace/<namespace name>` or keep the current namespace structure?  No, we should not. (yay!)
+
+     Keeping the current namespace structure mitigates changes, but prevents two orgs from having the same namespace and makes storing additional org-scoped resources like quota and groups more difficult.
+
+ 2.  Should we allow projects to be owned by Users or only allow ownership by Organizations?
+
+     Allowing Users to own projects eliminates the creation of another "linked" resource that needs to be protected and maintained.
+     Having a single entity allowed to own projects limits some duplication of things like quota.
+
+ 3.  Can we punt on adding users as organization members?
+
+     Organization-admins should not be able to force users into an Org.  Users should not be able to join any Org they want.  I'd like to get the structure correct and punt on the ability to add members.  It would require some sort of invitation/request/approval flow.
+
+ 4.  Can we punt on project transfering?
+
+     An org-owner shouldn't be able to take a project from someone else and they shouldn't be allowed to orphan their own project.  That means we'd need an offer/request/approval flow.
```

Comment:
> I'd see option for transferring ownership valuable. It usually helps with situations such as: ooh, I've made a mistake I want to change the owner of the project. 
> At the same time, I wouldn't worry about orphaning, since transferring you'd be required to give old and new owner.

Its currently described in a way that would allow a cluster-admin to transfer the projects.  That means its possible to build a way to do it, but I'd rather not try to provide out of the box.